src/components/DataTypes/enum.tsx

```diff
@@ -107,19 +108,30 @@ export const createFilter = (
 	return base;
 };
 
-export const Edit = ({
-	schema,
-	value,
-	onUpdate,
-	...props
-}: DataTypeEditProps & SelectProps & { slim?: boolean }) => (
-	<Select
-		{...props}
-		value={value}
-		onChange={(e: React.FormEvent<HTMLSelectElement>) =>
-			onUpdate(e.currentTarget.value)
-		}
-	>
-		{map(schema.enum, (item: string) => <option key={item}>{item}</option>)}
-	</Select>
-);
+type DataTypeSelectProps = DataTypeEditProps & SelectProps & { slim?: boolean };
```

Comment:
I'm guessing we want to consolidate the type declaration above and this one somewhere, but I'm not familiar enough with `rendition` to know where.